### PR TITLE
pr: Add back labels feature

### PR DESCRIPTION
**Pull request by hbons**

### CODE REVIEW

File: webviews/common/label.tsx [hunk 1]
```diff
@@ -14,18 +14,8 @@ export interface LabelProps {
...
-			style={{
-				backgroundColor: labelColor.backgroundColor,
-				color: labelColor.textColor,
-				borderColor: `${labelColor.borderColor}`,
-				paddingRight: canDelete ? '2px' : '8px'
-			}}
-		>
-			{name}{label.children}
-		</div>
+		<li>{name}{label.children}</li>
```
Comment (alexr00) [inline webviews/common/label.tsx:19]: I just remembered that this `Label` component is also used in the PR description (the full editor that shows all the details of an existing PR). We shouldn't make changes here that will also impact that view.

For now, can you just duplicate the `Label` component in this file and call it `LabelCreate`? Then we can safely modify `LabelCreate`.

**Review by alexr00 — CHANGES_REQUESTED** [review]

File: webviews/common/label.tsx (hunk 1 above)
Comment (hbons) [reply webviews/common/label.tsx:37]: Fixed.

**Review by alexr00 — DISMISSED** [review]
Looks good, feel free to merge when you're ready!

**Review by alexr00 — DISMISSED** [review]

**Comment by hbons** [comment]
Sorry, had to rebase again. :)

**Review by alexr00 — APPROVED** [review]

**Review by alexr00 — APPROVED** [review]

**Pull request closed without merging**